App.js

```diff
@@ -6,7 +6,7 @@ function MainContainer() {
       <colgroup className="MainContainer">
         <SideBar />
         <NewsFeedCol />
-        <TrendingCol/>
+        <TrendingCol />
```

Comment:
There is tooling available that can help you automatically format your code through a code editor like VSCode. [I recommend installing and configuring Prettier](https://prettier.io/).

Reply:
For VSCode I have Prettier already installed; this was done via CodeSandbox